src/Microsoft.DotNet.Wpf/src/PresentationCore/ref/PresentationCore.cs

```diff
@@ -2126,13 +2126,30 @@ public enum AutomationLiveSetting
         Polite = 1,
         Assertive = 2,
     }
+
+    public enum AutomationHeadingLevel 
```

Comment:
We should do the same thing WinUI/UWP does:  this public enum has values 0,1,2... which we map to the UIA ids 80050, 80051, ... before handing them off to UIA.

Reply:
There is an example in WPF where we make this kind of conversion between enums, so I can do it in the most correct way?

Reply:
The example was from WinUI, quoted in Ryland's email yesterday.

Reply:
I changed in the code of automation peer to do this mapping before returning the property value. Not sure if it is in the best place, my first tought was that it should be somewhere in UIAutomationClient, but when testing with Accessibility Insights, the best result was achieved with this implementation.

Reply:
AutomationPeer is the right place.